### DISCUSSION TITLE
Always open a copy Pull Request when source PR is update

### DISCUSSION THREAD

**Discussion by jbraffin** [Mergifyio/mergify#4088] [Q&A]
Hello,

Actually we have a workflow that copy pull requests and merge them in a target branch. However, sources pull requests are frequently update. When a copy pull request is merge, mergify never open another one with the updates. 

How can we proceed ? Is it possible ?

Best regards

**Comment by jd** [comment]
The best option for this is to use `backport`, which is essentially a copy expects that it is only run when the original PR is merged.

**Reply by jbraffin** [reply]
thank you for your reply. In fact target branch is use as a test with many different commit and deploy with CI/CD. Original PR is not merge before code is send in the test branch and this branch cannot be merge in the real original destination branch. So I think we couldn't use backport to do this, right ?

**Reply by jd** [reply]
No indeed, that wouldn't work.

I think what you want is to keep 2 branches in sync (the PR head branches) with 2 different bases (e.g., `staging` vs `prod`). This is not something Mergify solves right now.
